lib/SPIRV/SPIRVToOCL.cpp

```diff
@@ -440,6 +445,23 @@ Instruction *SPIRVToOCL::visitCallSPIRVAtomicBuiltin(CallInst *CI, Op OC) {
 } // namespace SPIRV
 
 ModulePass *llvm::createSPIRVToOCL(Module &M) {
+  if (OCLBuiltinsVersion.getNumOccurrences() > 0) {
+    if (OCLBuiltinsVersion.getValue() == "CL1.2")
+      return createSPIRVToOCL12();
+    else if (OCLBuiltinsVersion.getValue() == "CL2.0" ||
+             OCLBuiltinsVersion.getValue() == "CL2.1")
```

Comment:
Note: there is no such thing as OpenCL C 2.1

Reply:
Right. Fixed.